rcon/squad_automod/seeding_rules.py

```diff
@@ -104,10 +107,10 @@ def punitions_to_apply(self, _, squad_name: str, team: str, squad: dict) -> Puni
                 )
 
                 violations = []
-                print(self.config.disallowed_roles.roles)
-                if aplayer.role in self.config.disallowed_roles.roles:
-                    violations.append(self.config.disallowed_roles.message.format(
-                        role=self.config.disallowed_roles.roles.get(aplayer.role)))
+                if self.config.disallowed_roles.min_players < server_player_count < self.config.disallowed_roles.max_players:
```

Comment:
This is not an inclusive lower bound as described in the config.

Reply:
Good catch! I'll look into this with the next commits 👍